stable/insights-agent/templates/test/test-deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
           env:
           {{ include "proxy-env-spec" . | indent 10 | trim }}
           securityContext:
-            {{- toYaml .Values.securityContext | nindent 12 }}
+            {{- toYaml .Values.test.securityContext | nindent 12 }}
```

Comment:
Do we want this new value included in values.yaml for documentation? I see it does have other `test.*` values there.

Reply:
Good question... I can add that in the documentation.